Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This file describes the standard way to build libcompose, using docker
-FROM golang:1.4.2-cross
+FROM golang:1.5.1
```

Comment:
I think `golang:1.5` was actually the better way to go. If you look at https://hub.docker.com/_/golang/ they're both the same image right now. But if 1.5.2 gets released, we'd be auto-upgraded to use it.

Unless we expect a minor release to break something, it's probably easier to go with the more general tag.

This is minor though, I'm ok either way.